default/stringtables/en.txt

```diff
@@ -14194,16 +14192,16 @@ ULTIMATE_SUPPLY_DESC
 ++	Ultimate [[metertype METER_SUPPLY]]: +3
 
 NO_STOCKPILE_DESC
---	No [[metertype METER_STOCKPILE]]
+−−	No [[metertype METER_STOCKPILE]]
 
 BAD_STOCKPILE_DESC
-	Bad [[metertype METER_STOCKPILE]]: 50%
+−	Bad [[metertype METER_STOCKPILE]]: 50%
 
 AVERAGE_STOCKPILE_DESC
-	Average [[metertype METER_STOCKPILE]]: 100%
+'''	Average [[metertype METER_STOCKPILE]]: 100%'''
```

Comment:
?

Reply:
To make it clear there is a tab space at the beginning. The three quotes are used for each "Average [meter]" strings (e.g. AVERAGE_SUPPLY_DESC a few lines before), except this one. So I added it.